8.x/theme-development/README.md

```diff
@@ -4,8 +4,130 @@ pageClass: no-toc
 
 ## :rocket: 5 minute quick start
 
-Larammerce is using the Laravel framework as its core system and obviously inherits all Laravel features. for example, Larammerce uses Blade (Laravel’s templating engine), to easily create a standard page template, or layout.
+this document teaches you How to install the larammerce-base-theme project from GitHub
```

Comment:
Please note that all the sentences must begin with capital letters,

this document -> This document
also rewrite your paragraph like this:
This document is about installing the Larammerce base theme on the Larammerce platform and it's a good starting point for beginners to learn how to setup and customize their own themes.